### ISSUE TITLE
New generative ring method

### ISSUE THREAD

**Issue by GuillemGongora** [tmhglnd/mercury#59]
Add new generative ring method similar to fill() that repeats every value in a ring a certain amount of times.
ring abc (1 2 3 4) -> (1 2 3 4)
ring cba newMethod(abc 4) -> (1 1 1 1 2 2 2 2 3 3 3 3 4 4 4 4)

**Comment by tmhglnd** [comment]
Ah that is a nice suggestions! Currently a way to do this is:

```java
ring vals [1 2 3 4]
ring reps lace(vals vals vals vals)
print vals
//=> [ 1 1 1 1 2 2 2 2 3 3 3 3 4 4 4 4 ]
```

**Comment by GuillemGongora** [comment]
Perfect! But it's a problem when working with big complex rings like:

`ring offsets spreadInclusiveFloat(16 0.945 0.985)`
`ring offsets invert(offsets)`
`ring offsets lace(offsets offsets offsets offsets)`

Cannot do this with `fill()`

**Comment by tmhglnd** [comment]
Yeah exactly. Added it to my to do list for the total-serialism package: https://github.com/tmhglnd/total-serialism/issues

**Comment by tmhglnd** [comment]
Now included in e105a4eebdaa2f6f39342545822dc7ef44bf5c85
